Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -476,11 +476,6 @@ WI.contentLoaded = function()
         WI._dockedResizerElement.addEventListener("mousedown", WI._handleDockedResizerMouseDown);
     }
 
-    if (supportsUndocked) {
-        let undockedTitleAreaElement = document.getElementById("undocked-title-area");
```

Comment:
might be worth checking with other platforms to make sure this wont cause issues

im pretty sure it's fine since for everything other than macOS `--undocked-title-area-height: 0px;` but couldn't hurt to make sure